int64 speed_percent
int8 is_reverse
---
int64 error
